public/assets/js/main.js

```diff
@@ -1,10 +1,11 @@
-(function() {
+(function mainJS() {
```

Comment:
There is an ESLint for this, it's just that it results in so many warnings.

I'll make a PR some day with some tweaks and ping you.